homeassistant/components/sensor/qnap.py

```diff
@@ -99,6 +101,7 @@
     vol.Optional(CONF_NICS): cv.ensure_list,
     vol.Optional(CONF_DRIVES): cv.ensure_list,
     vol.Optional(CONF_VOLUMES): cv.ensure_list,
+    vol.Optional(CONF_ALLOW_UNREACHABLE, default=False): cv.boolean,
```

Comment:
Why you make that as option?

Reply:
hi, the reasoning is that if this setting is False (the default), this sensor keeps its old behaviour. This is for people who have their NAS on 24/7 and want to see the notification. Setting it to True hides the notification and uses the HA PlatformNotReady mechanism.

BUT: maybe this option should dissapear and the PlatformNotReady should become the default, ditching the notification, that's another option and I'm open to opinions on this.

Reply:
It should be only `PlatformNotReady` and the should be covered/handled by `available()`. Adding an option to configure the behavior of the notification looks wired .

Reply:
Hi, ok If I understand correctly you would just drop the config option and the notification, and use PlatformNotReady as the only behaviour. Sounds good, I'll remove the config. Anything else that needs to be changed? Not familiar with "available()".

Reply:
Available property is simple: return False if the the device is not reachable (maybe the state fetch going wrong on update) and True if they is back.